.github/workflows/release_update_readme.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: update-readme-on-release
+on:
+  release:
+    types: [published]
+permissions:
+  contents: read
+jobs:
+  update-readme:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3  # v3.5.0
+      with:
+        ref: main
+    - name: Import GPG key
+      shell: bash
+      env:
+        BUILDBOT_GPG_KEY_B64: ${{ secrets.BUILDBOT_GPG_KEY_B64 }}
+        BUILDBOT_GPG_KEY_ID: ${{ secrets.BUILDBOT_GPG_KEY_ID }}
+      run: |
+        echo "${BUILDBOT_GPG_KEY_B64}" | base64 --decode | gpg --no-tty --batch --import
+        git config --global user.signingkey "${BUILDBOT_GPG_KEY_ID}"
+        git config --global commit.gpgsign true
+    - name: Import SSH key
+      shell: bash
+      env:
+        BUILDBOT_SSH_KEY_B64: ${{ secrets.BUILDBOT_SSH_KEY_B64 }}
+      run: |
+        echo "${BUILDBOT_SSH_KEY_B64}" | base64 --decode > /tmp/ssh.key
+        chmod 600 /tmp/ssh.key
+    - name: Setup git
+      shell: bash
+      env:
+        GIT_SSH_COMMAND: "ssh -i /tmp/ssh.key"
+      run: |
+        git config --global user.name 'pixie-io-buildbot'
+        git config --global user.email 'build@pixielabs.ai'
+        git remote add fork git@github.com:pixie-io-buildbot/pixie.git
+    - name: Update readme locally
+      shell: bash
+      env:
+        TAG_NAME: ${{ github.event.release.tag_name }}
+        URL: ${{ github.event.release.html_url }}
+      run: |
+        export COMPONENT_VERSION="${TAG_NAME#release/}"
+        export COMPONENT="${COMPONENT_VERSION%%/*}"
+        export VERSION="${COMPONENT_VERSION#*/}"
```

Comment:
I wonder if using `cut -d "/"` is more readable than bash string manipulation ...